### PR TITLE
Front matter

### PR DESCRIPTION
**Pull request by dwillmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,84 @@
-# phosphor-command
+phosphor-command
+================
+
+[![Build Status](https://travis-ci.org/phosphorjs/phosphor-commands.svg)](https://travis-ci.org/phosphorjs/phosphor-command?branch=master)
+[![Coverage Status](https://coveralls.io/repos/phosphorjs/phosphor-commands/badge.svg?branch=master&service=github)](https://coveralls.io/github/phosphorjs/phosphor-command?branch=master)
+
 A module for expressing the command pattern.
+
+[API Docs](http://phosphorjs.github.io/phosphor-commands/api/)
```
Comment (blink1073) [inline README.md]: command*

File: README.md [hunk 2]
```diff
@@ -1,2 +1,84 @@
-# phosphor-command
+phosphor-command
+================
+
+[![Build Status](https://travis-ci.org/phosphorjs/phosphor-commands.svg)](https://travis-ci.org/phosphorjs/phosphor-command?branch=master)
```
Comment (sccolbert) [inline README.md]: s/commands/command

Comment (blink1073) [inline README.md]: command.svg

File: README.md [hunk 3]
```diff
@@ -1,2 +1,84 @@
...
+[![Coverage Status](https://coveralls.io/repos/phosphorjs/phosphor-commands/badge.svg?branch=master&service=github)](https://coveralls.io/github/phosphorjs/phosphor-command?branch=master)
```
Comment (sccolbert) [inline README.md]: s/commands/command

File: README.md (hunk 1 above)
Comment (sccolbert) [inline README.md]: s/commands/command

File: README.md [hunk 4]
```diff
@@ -1,2 +1,84 @@
...
+- Chrome 38+
+
+Bundle for the Browser
+----------------------
+
+Follow the package install instructions first.
+
+```bash
+npm install --save-dev browserify browserify-css
+browserify -g browserify-css myapp.js -o mybundle.js
```
Comment (blink1073) [inline README.md]: browserify-css no longer needed here.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    "clean": "rimraf lib && rimraf test/build && rimraf test/coverage",
+    "build:src": "tsc --project src",
+    "build:test": "tsc --project test/src",
+    "build": "npm run build:src && npm run build:test",
+    "docs": "rimraf docs/api && typedoc --options scripts/tdoptions.json",
+    "prepublish": "npm run build",
+    "test": "mocha test/build/index.js",
+    "test:coverage": "istanbul cover --dir test/coverage node_modules/mocha/bin/_mocha -- test/build/index.js"
+  },
+  "repository": {
```
Comment (blink1073) [inline package.json]: add "browserify": {} entry here like phopshor-widget

File: README.md [hunk 6]
```diff
@@ -1,2 +1,84 @@
...
+- Firefox 32+
+- Chrome 38+
+
+Bundle for the Browser
+----------------------
+
+Follow the package install instructions first.
+
+```bash
+npm install --save-dev browserify browserify-css
```
Comment (sccolbert) [inline README.md]: Readme's of projects not requiring CSS were updated yesterday to this:
```
npm install --save-dev browserify
browserify myapp.js -o mybundle.js
```

File: package.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    "url": "https://github.com/phosphorjs/phosphor-command.git"
+  },
+  "files": [
+    "lib/index.js",
+    "lib/index.d.ts"
+  ],
+  "keywords": [
+    "command",
+    "commands",
+    "manager"
```
Comment (sccolbert) [inline package.json]: no "manager" keyword

File: package.json (hunk 5 above)
Comment (blink1073) [inline package.json]: Cancel that, css not needed in this repo.

**Pull request merged**